vignettes/admiral.Rmd

```diff
@@ -19,135 +19,69 @@ library(admiraldev)
 # Main Idea
 
 The main idea of `{admiral}` is that an ADaM dataset is built by a sequence of
-derivations. Each derivation adds one or more variables or parameters to the
+derivations. Each derivation adds one or more variables or records to the
 processed dataset. This modular approach makes it easy to adjust code by adding,
-removing, or modifying derivations. Each derivation is a function call. Consider
-for example the following script which creates a (very simple) ADSL dataset.
+removing, or modifying derivations. Each derivation is a function call. 
 
-## Load Packages and Example Datasets
-
-First, we will load our packages and example datasets to help with our `ADSL` creation. The `{dplyr}` and `{lubridate}` packages are `{tidyverse}` packages and used heavily throughout this script.  The `{admiral}` package also leverages the `{pharmaversesdtm}` package for example SDTM datasets which are from the CDISC Pilot Study.
+In this vignette we will explore some of the different type of derivation functions 
+offered by `{admiral}`, as well as the argument conventions they follow and how best 
+to start an `{admiral}` script. To help with the examples showcased, we will load 
+some packages and set up some example data to manipulate later. The `{dplyr}` and 
+`{lubridate}` and `{stringr}` packages are `{tidyverse}` packages and are used heavily
+throughout this vignette.  The `{admiral}` package also leverages the `{pharmaversesdtm}` 
+package for example SDTM datasets which are from the CDISC Pilot Study.
 
 ```{r, message=FALSE, warning=FALSE}
 library(dplyr, warn.conflicts = FALSE)
```

Comment:
Do we want a commented out install.packages(c("dplyr", .... , "admiral") so a user could quickly install every package ?